Bring Lift and Control Panel code back

bot190 opened the pull request.

bot190 committed:
Bring Lift and Control Panel code back

msoucy commented on src/main/java/frc/robot/Robot.java:
```diff
@@ -202,16 +210,28 @@ public CommandBase shootAtSpeed(final int ballAmount, double shooterSpeed) {
     }
 
+    public CommandBase afterMatchPit() {
+        final CommandBase cmd = new SequentialCommandGroup(lift.resetLift(), intake.retractIntake());
```
Pssst. We can probably condense these to one line with `return sequential("Name", foo, bar);`

msoucy commented on src/main/java/frc/robot/Robot.java:
```diff
@@ -202,16 +210,28 @@ public CommandBase shootAtSpeed(final int ballAmount, double shooterSpeed) {
     public CommandBase cancelAll() {
         final CommandBase cmd = new ParallelCommandGroup(drive.cancel(), indexer.cancel(), intake.cancel(),
-                shooter.cancel());
+                lift.cancel(), shooter.cancel());
```
I can't wait until this is automated... #reflection

msoucy commented on src/main/java/frc/robot/maps/FrancoisMap.java:
```diff
@@ -28,7 +28,7 @@
     // controlPanel is defined here due to the gyro being plugged into this speed
     // controller as well as the control panel motor
-    WPI_TalonSRX gyro = new WPI_TalonSRX(43);
+    WPI_TalonSRX controlPanel = new WPI_TalonSRX(43);
```
Much better name.

msoucy reviewed (changes requested):
We definitely want to test/review this

bot190 committed:
Bring back the old indexer code with singulator control

bot190 committed:
Bring back copilot controls

bot190 committed:
Simplify commands in Robot.java

use sequence() and parallel helpers where possible. Otherwise use .withName() in robot.java. Other files still need to be updated.

bot190 committed:
Latest chopshoplib bring back index sensors
